flyteplugins/go/tasks/plugins/k8s/kfoperators/mpi/mpi.go

```diff
@@ -151,37 +125,19 @@ func (mpiOperatorResourceHandler) BuildResource(ctx context.Context, taskCtx plu
 			"Invalid TaskSpecification, unsupported task template version [%v] key", taskTemplate.TaskTypeVersion)
 	}
 
-	if workerReplica.ReplicaNum == 0 {
+	if *workerReplicaSpec.Replicas <= 0 {
 		return nil, fmt.Errorf("number of worker should be more then 0")
```

Comment:
worth fixing the typos? number of worker should be more then -> number of workers should be greater than